corehq/apps/app_manager/models.py

```diff
@@ -818,6 +818,7 @@ class FormBase(DocumentSchema):
     form_links = SchemaListProperty(FormLink)
     schedule_form_id = StringProperty()
     custom_instances = SchemaListProperty(CustomInstance)
+    case_references_data = DictProperty()
```

Comment:
Also I really want to define a schema for this instead of using a DictProperty. I don't think there are any reasons not to do that but I didn't get around to it.